<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContrastWizardPageForm</class>
 <widget class="QWizardPage" name="ContrastWizardPageForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contrast</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_18">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_17">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_13">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Type</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QComboBox" name="m_contrastTypeComboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Raster Bands</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_24">
          <item row="0" column="0">
           <widget class="QTableWidget" name="m_bandTableWidget">
            <property name="alternatingRowColors">
             <bool>false</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <property name="showGrid">
             <bool>true</bool>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderMinimumSectionSize">
             <number>29</number>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Band</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="m_previewGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Preview</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_19">
        <item row="0" column="0">
         <widget class="QFrame" name="m_frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="m_histogramPushButton">
          <property name="text">
           <string>Histogram...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_previewGroupBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
